### PR TITLE
Fix service guide URLs across several packages

### PR DESCRIPTION
**Pull request by nickbp**
Update to latest URL scheme. Remove/fix URL in cases where the URL
doesn't exist or is incorrect. (sloppy copy/paste job?)

### CODE REVIEW

**Comment by jsancio** [comment]
LTGM!

File: repo/packages/H/hdfs/5/package.json [hunk 1]
```diff
@@ -14,6 +14,6 @@
   ],
   "preInstallNotes": "In order for HDFS to start successfully, it requires a minimum of five nodes; each with at least 2 CPU shares and 8GB of RAM available for use by the HDFS Service.\nNote that the service is experimental and there may be bugs, including possible data loss, incomplete features, incorrect documentation or other discrepancies.",
   "postInstallNotes": "DC/OS HDFS Service is being installed.\n\n\tDocumentation: https://docs.mesosphere.com/1.7/usage/service-guides/hdfs/\n\tIssues: https://dcosjira.atlassian.net/projects/HDFS/issues",
-  "postUninstallNotes": "DC/OS HDFS Service has been uninstalled.\nPlease follow the instructions at http://docs.mesosphere.com/services/hdfs/#uninstall to remove any persistent state if required.",
+  "postUninstallNotes": "DC/OS HDFS Service has been uninstalled.\nPlease follow the instructions at http://docs.mesosphere.com/1.7/usage/service-guides/hdfs/#uninstall to remove any persistent state if required.",
```
Comment (joel-hamill) [inline repo/packages/H/hdfs/5/package.json]: Do you specifically want the 1.7 doc, or do you want current?

Comment (nickbp) [inline repo/packages/H/hdfs/5/package.json]: I had checked, it looks like there aren't 1.8 versions of the docs yet (/current/ => 404)

Comment (joel-hamill) [inline repo/packages/H/hdfs/5/package.json]: This `https://docs.mesosphere.com/usage/service-guides/hdfs/` and this `https://docs.mesosphere.com/usage/services/hdfs/` (defaults to current) are working for me.

Comment (nickbp) [inline repo/packages/H/hdfs/5/package.json]: Oh interesting, I must have typo'ed it earlier when testing it out. I'll update this to use `current`.

Comment (nickbp) [inline repo/packages/H/hdfs/5/package.json]: Done

**Comment by triclambert** [comment]
Infinity changes lgtm.

**Pull request merged**